client/src/elements/ImageElement/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Image } from 'semantic-ui-react';
+
+const ImageElement = (props) => (
```

Comment:
Do we really need this component? It looks like an unnecessary wrapper.

Reply:
this is semantics image wrapper, it works a little differently, so we do need a wrapper.